--- opencart/opencart/upload/system/config/default.php	(date 1526887930446)
+++ opencart/opencart/upload/system/config/default.php	(date 1526887930446)
@@ -1,7 +1,7 @@
 <?php
 // Site
 $_['site_base']            = '';
-$_['site_ssl']             = false;
+$_['site_ssl']             = SITE_SSL;
 
 // Language
 $_['language_default']     = 'en-gb';

--- opencart/opencart/upload/system/config/catalog.php	(date 1526893511731)
+++ opencart/opencart/upload/system/config/catalog.php	(date 1526893511731)
@@ -1,7 +1,7 @@
 <?php
 // Site
 $_['site_base']        = substr(HTTP_SERVER, 7);
-$_['site_ssl']         = false;
+$_['site_ssl']         = SITE_SSL;

 // Database
 $_['db_autostart']     = true;
