Index: vendor/byjg/migration/src/Migration.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- vendor/byjg/migration/src/Migration.php	(date 1522660605000)
+++ vendor/byjg/migration/src/Migration.php	(date 1522660605000)
@@ -265,18 +265,27 @@
             throw new DatabaseIsIncompleteException('Database was not fully updated. Use --force for migrate.');
         }
 
-        while ($this->canContinue($currentVersion, $upVersion, $increment)
-            && file_exists($file = $this->getMigrationSql($currentVersion, $increment))
-        ) {
-            if ($this->callableProgress) {
-                call_user_func_array($this->callableProgress, ['migrate', $currentVersion]);
-            }
+        $this->getDbDriver()->beginTransaction();
+        try {
+
+            while ($this->canContinue($currentVersion, $upVersion, $increment)
+                && file_exists($file = $this->getMigrationSql($currentVersion, $increment))
+            ) {
+                if ($this->callableProgress) {
+                    call_user_func_array($this->callableProgress, ['migrate', $currentVersion]);
+                }
 
-            $this->getDbCommand()->setVersion($currentVersion, 'partial ' . ($increment>0 ? 'up' : 'down'));
-            $this->getDbCommand()->executeSql(file_get_contents($file));
-            $this->getDbCommand()->setVersion($currentVersion, 'complete');
-            $currentVersion = $currentVersion + $increment;
+                $this->getDbCommand()->setVersion($currentVersion, 'partial ' . ($increment > 0 ? 'up' : 'down'));
+                $this->getDbCommand()->executeSql(file_get_contents($file));
+                $this->getDbCommand()->setVersion($currentVersion, 'complete');
+                $currentVersion = $currentVersion + $increment;
+            }
+            $this->getDbDriver()->commitTransaction();
         }
+        catch (\Throwable $e){
+            $this->getDbDriver()->rollbackTransaction();
+            throw $e;
+        }
     }
 
     /**
